05-JWT-Basics/starter/errors/unauthenticated.js

```diff
@@ -0,0 +1,13 @@
+const CustomAPIError = require("./custom-error");
+const { StatusCodes } = require("http-status-codes");
+
+
+class UnauthenticatedError extends CustomAPIError {
+  constructor(message) {
+    super(message);
+    //this.statusCode = 401
+    this.statusCode = StatusCodes.UNAUTHORIZED;
```

Comment:
As a learning exercise, it's definitely worthwhile to just look up the response code you need until you remember it; there are only a few popular ones (200, 201, 400, 403, 404, 500 are byfar most common.

I think that in a lot of company code, for better or worse, they're likely going to hard code response codes in a lot of places.